src/test/java/jenkins/plugins/git/GitBranchSCMHeadTest.java

```diff
@@ -24,18 +24,10 @@
     public JenkinsRule j = new JenkinsRule() {
         @Override
         public void before() throws Throwable {
-            if (!Functions.isWindows() && "testMigrationNoBuildStorm".equals(this.getTestDescription().getMethodName())) {
+            if ("testMigrationNoBuildStorm".equals(this.getTestDescription().getMethodName())) {
                 URL res = getClass().getResource("/jenkins/plugins/git/GitBranchSCMHeadTest/testMigrationNoBuildStorm_repositories.zip");
-                final File path = new File("/tmp/JENKINS-48061");
-                if (path.exists()) {
-                    if (path.isDirectory()) {
-                        FileUtils.deleteDirectory(path);
-                    } else {
-                        path.delete();
-                    }
-                }
-
-                new FilePath(new File(res.toURI())).unzip(new FilePath(path.getParentFile()));
+                final File path = Files.createTempDirectory("JENKINS-48061").toFile();
```

Comment:
For this to work we will need to have two different `@LocalData` for windows and unix systems, since the job and shared library are configured to `/tmp/JENKINS-48061` and env variable resolution wasn't easily achieved.